styles.css

```diff
@@ -0,0 +1,176 @@
+body {
+  font-family: "Josefin Sans";
+  background-color: #fffafb;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  scroll-behavior: smooth;
+}
+.instruction {
+  position: absolute;
+  left: 100px;
+}
+
+.logo img {
+  width: 15%;
+}
+header {
+  position: fixed;
+  background-color: #f4f2f2;
+  top: 0;
+  height: 100px;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  min-height: 75px;
+  padding: 0px 20px;
+  font-size: 20px;
+}
+nav {
+  position: absolute;
+  margin-top: 10px;
+  margin-right: 0px;
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  align-items: center;
+  text-align: center;
+  padding: 0 50px;
+}
+nav a {
+  display: inline-block;
+  background-color: #852fb430;
+  width: 10%;
+  font-size: 20px;
+  padding: 8px 8px;
+  text-align: center;
+  margin: 10px;
+  color: #000000;
+  border-radius: 5px;
+  text-decoration: none;
+  overflow: auto;
+}
+
+nav a:hover {
+  background-color: purple;
+  color: #fff;
+}
+.grid {
+  display: flex;
+  font-size: 25px;
+  align-items: center;
+  margin-left: 200px;
+  margin-top: 30px;
+}
+#features {
+  margin-top: 150px;
+  width: 80%;
+}
+
+.icon {
+  color: #9b0b838d;
+  width: 150px;
+  align-items: center;
+}
+
+#pricing {
+  display: flex;
+  text-align: center;
+  justify-content: center;
+  font-size: 25px;
+  margin-top: 60px;
+  gap: 60px;
+}
+
+.product {
+  padding: 30px;
+  border: 2px solid black;
+}
+
+.btn {
+  background-color: hwb(279 2% 64% / 0.567);
+  padding: 5px 10px;
+  width: 100px;
+}
+
+.btn:hover {
+  background-color: purple;
+  color: #fff;
+}
+
+ol {
+  list-style: none;
+}
+
+#how-to-use {
+  margin-left: 450px;
+  margin-top: 100px;
+}
+
+#how-to-use > h2 {
+  margin-left: 300px;
+  margin-bottom: 30px;
+}
+
+.ship {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin-top: 50px;
+  height: 600px;
+  font-size: 25px;
+}
+
+.ship #purchase #form h2 {
+  margin-bottom: 25px;
+}
```

Comment:
- [ ] You have tried you best to suggest the changes and provided `the importance`👍 . However, the required changes may not be very clear to the student. This is why in your explanaition, you should ensure to be more detailed by explaining `why` and `how important`. Therefore i suggest that in your comments you add consider the 2 elements of `why` and `how important` ☺️

Reply:
I suggest you give space before writing the next selector. This will make your code more readable and you will be able to quickly detect which selector you want to locate. It is very important because it will give you a linters error 😊 .